vm/message/Cargo.toml

```diff
@@ -23,13 +23,18 @@ fvm               = "1.1"
 fvm_ipld_encoding = "0.2"
 fvm_shared        = { version = "0.8", default-features = false, features = ["testing"] }
 num-traits        = "0.2"
+quickcheck        = { version = "1" }
 serde             = { version = "1.0", features = ["derive"] }
 
 [dev-dependencies]
-serde_json = "1.0"
+quickcheck_macros = "1"
+serde_json        = "1.0"
 
 [features]
 default = ["blst"]
 blst    = ["forest_crypto/blst", "fvm_shared/blst"]
 
 proofs = []
+
+[package.metadata.cargo-udeps.ignore]
+normal = ["quickcheck"]
```

Comment:
Remove this.

Reply:
The PR was failing the unused dependencies check without this (`cargo udeps` apparently does not see dependencies used only in tests?).

Reply:
If a dependency is only used for testing, it should be in `dev-dependencies`.

Reply:
Suppressing warnings is a last resort and you should always ask in #fil-devs before doing it.

Reply:
Ah but I see that one is not required.

Reply:
I don't think we can put this one under `dev-dependencies` though. See the discussion under PR #1879 .

Reply:
Have you tried?

Reply:
I guess I didn't since I was just following the example. Seems to work!